terraform/main.tf

```
@@ -220,6 +220,13 @@ module "test-docs-hosting" {
   domain_map       = local.domain_map
 }
 
+module "test-pypi-camo" {
+  source = "./image-proxy"
+
+  sitename             = "Test PyPI Camo"
+  domain               = "testpypi-image-proxy.global.ssl.fastly.net"
```

I think this is going to conflict with the service you created manually to test.

I suggest we rename this:

```suggestion
  domain               = "testpypi-camo.global.ssl.fastly.net"
```